conda/meta.yaml

```diff
@@ -45,9 +45,8 @@ requirements:
 
 about:
   home: "https://sdv.dev"
-  license: MIT
```

Comment:
I think we can just delete this file. We don't do these conda releases anymore

Reply:
+1 we could just delete the entire `conda/` folder while we're at it right? Seems we did this for [Copulas](https://github.com/sdv-dev/Copulas) already.